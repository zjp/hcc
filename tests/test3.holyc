@^
[]
{}
()
;,
---
+++*
/!
==!=
<>
&<=
>==
&&||
2
int intptr bool boolptr char charptr
void if else while return true false
FROMCONSOLE TOCONSOLE NULLPTR
