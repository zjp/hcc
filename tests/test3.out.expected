AT [1,1]
CARAT [1,2]
LBRACE [2,1]
RBRACE [2,2]
LCURLY [3,1]
RCURLY [3,2]
LPAREN [4,1]
RPAREN [4,2]
SEMICOLON [5,1]
COMMA [5,2]
DASHDASH [6,1]
DASH [6,3]
CROSSCROSS [7,1]
CROSS [7,3]
STAR [7,4]
SLASH [8,1]
NOT [8,2]
EQUALS [9,1]
NOTEQUALS [9,3]
LESS [10,1]
GREATER [10,2]
AMP [11,1]
LESSEQ [11,2]
GREATEREQ [12,1]
ASSIGN [12,3]
AND [13,1]
OR [13,3]
INTLIT:2 [14,1]
INT [15,1]
INTPTR [15,5]
BOOL [15,12]
BOOLPTR [15,17]
CHAR [15,25]
CHARPTR [15,30]
VOID [16,1]
IF [16,6]
ELSE [16,9]
WHILE [16,14]
RETURN [16,20]
TRUE [16,27]
FALSE [16,32]
FROMCONSOLE [17,1]
TOCONSOLE [17,13]
NULLPTR [17,23]
EOF [18,1]
